File: python/bc2031.py
res = ""
count = int(input())

for c in range(0, count):
    fPl = input()
    sPl = input()

    if fPl == "pedra":
        if sPl == "ataque": res = "Jogador 2 venceu"
        elif sPl == "papel": res = "Jogador 1 venceu"
        else: res = "Sem ganhador"
    elif fPl == "papel":
        res = "Jogador 2 venceu" if sPl == "pedra" or sPl == "ataque" else "Ambos venceram"
    else:
        res = "Jogador 1 venceu" if sPl == "pedra" or sPl == "papel" else "Aniquilacao mutua"
    print(res)

File: cpp/bc2031.cpp
#include <iostream>

using namespace std;

int main()
{
	string fPl, sPl, res;
	int count;
	
	cin >> count;
	
	for(int i = 0; i < count; i++)
	{
		
		cin >> fPl;
		cin >> sPl;
		
		if(fPl == "pedra")
		{
			if(sPl == "ataque") res = "Jogador 2 venceu";
			else if(sPl == "papel") res = "Jogador 1 venceu";
			else res = "Sem ganhador";
		}
		else if(fPl == "papel")
			sPl == "pedra" || sPl == "ataque"? res = "Jogador 2 venceu" : res ="Ambos venceram";
		else
			sPl == "pedra" || sPl == "papel"? res ="Jogador 1 venceu": res = "Aniquilacao mutua";
		
		cout << res << endl;
	}
	
}
